crates/core/src/pattern/patterns.rs

```diff
@@ -482,7 +483,7 @@ impl Pattern {
             }
             if node_types[sort as usize].is_empty() {
                 let content = node.utf8_text(text)?;
-                if node.named_child_count() == 0
+                if (node.named_child_count() == 0 || node.named_child_count() == 1)
```

Comment:
Why this change?

Reply:
I just reverted this change, it was related to my attempt to match partial nodes.